### PR TITLE
Add simpleclient_httpserver

### PR DESCRIPTION
**Pull request by tgulacsi**
For the simplest HTTP server, without extra dependencies.
Uses com.sun.net.httpserver.HttpServer, which is in JRE 1.6.

### CODE REVIEW

File: simpleclient_httpserver/pom.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            <version>0.0.9-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>io.prometheus</groupId>
+            <artifactId>simpleclient_common</artifactId>
+            <version>0.0.9-SNAPSHOT</version>
+        </dependency>
+        <!-- Test Dependencies Follow -->
+        <dependency>
+            <groupId>junit</groupId>
```
Comment (brian-brazil) [inline simpleclient_httpserver/pom.xml]: You aren't using junit, can you remove this?

File: simpleclient_httpserver/src/main/java/io/prometheus/client/exporter/HTTPServer.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import java.io.OutputStreamWriter;
+import java.util.Enumeration;
+import java.util.concurrent.Executors;
+
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import com.sun.net.httpserver.HttpExchange;
+
+public class HTTPServer {
+	static class HTTPMetricHandler implements HttpHandler {
```
Comment (brian-brazil) [inline simpleclient_httpserver/src/main/java/io/prometheus/client/exporter/HTTPServer.java]: Could you use 4 spaces rather than tabs?

**Comment by brian-brazil** [comment]
That generally looks good, can you update the README?

**Comment by robsonpeixoto** [comment]
News about it?

**Comment by brian-brazil** [comment]
Superseded by #276

**Pull request closed without merging**